### ISSUE TITLE
Bug: version.json generation is broken because release tags are missing

### ISSUE THREAD

**Issue by mistressofjellyfish** [mistressofjellyfish/not-grocy#20]
When I transferred the files of this repo, I pushed the history but forgot to push the tags. This means that `git describe --tags` will fail to find a tag on a fresh clone.

To fix:

- use `git push --tags` to re-push all tags from the original repo
- create one tag on the commit this project was forked.

**Comment by mistressofjellyfish** [comment]
Tags are back again, now I have to figure out where to best place the "forked" tag.

There are two possibilities: f7bc6a3 or 41ca7a2. I'm leaning towards f7bc6a3, because all those subsequent commits aren't there in the original projects repository.

**Comment by mohe2015** [comment]
I would agree. Also does this tag have any significance except being the in the version number while there isn't a new release?

**Comment by mistressofjellyfish** [comment]
At the moment, it does not.
